admin-node-setup.sh

```diff
@@ -61,6 +61,10 @@ fi
 # Generate TLS CA and Initial Certificates
 /usr/share/caasp-container-manifests/gen-certs.sh
 
+# add an entry for ldap.infra.caasp.local to /etc/hosts
+# this is needed to enable net-ldap to validate the certificate for LDAP_HOST
+sed -i 's/127.0.0.1\tlocalhost/127.0.0.1\tlocalhost ldap.infra.caasp.local/g' /etc/hosts
```

Comment:
We'll need to make this a little more robust - if ran over and over (as `admin-node-setup.sh` is on each reboot), we'll end up adding this new entry multiple times.

Reply:
We'll also want to update salt's salt/etc-hosts/hosts.jinja to make sure that doesn't remove this entry from the admin node.

Reply:
Something like this maybe?

	if [ "$(cat /etc/hosts | grep -E "^127.0.0.1\s+" | grep ldap.infra.caasp.local)" ]; then
	    echo "Have it already"
	else
	    echo "Don't have it"
	fi

Reply:
will we still need this after bootstrap? i am not fully sure, but for now i think it's only needed when creating the ldap user, but i might be wrong

Reply:
Yes, the Velum login for now connects to the only internal LDAP using the `LDAP_HOST` envvar, so the `/etc/hosts` needs to resolve whatever that is.